concourse/pipelines/replicator.py

```diff
@@ -301,9 +303,15 @@ def deploy(self, definition_descriptor):
                 api.unpause_pipeline(pipeline_name=pipeline_name)
             if self.expose_pipelines:
                 api.expose_pipeline(pipeline_name=pipeline_name)
+
+            if response is client.SetPipelineResult.CREATED:
```

Comment:
use OR for the deploy_status

always:  deploy_status = DeployStatus.SUCCEEDED
if if response is client.SetPipelineResult.CREATED:
  deploy_status |= deploy_status = DeployStatus.CREATED